tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -243,65 +106,6 @@ def test_post_sms_notification_with_celery_persistence_returns_201_with_sms_send
     assert mocked.called
 
 
-def test_post_sms_notification_uses_sms_sender_id_reply_to(notify_api, client, sample_template_with_placeholders, mocker):
```

Comment:
We might want to change this test instead of axing it, by mocking the celery database-task and checking what it receives maybe?